diff --git a/Source/WTF/wtf/StackBounds.h b/Source/WTF/wtf/StackBounds.h
index 13f6be629..ceaa38299 100644
--- a/Source/WTF/wtf/StackBounds.h
+++ b/Source/WTF/wtf/StackBounds.h
@@ -145,7 +145,7 @@ private:
 
     void checkConsistency() const
     {
-#if !ASSERT_DISABLED
+#if 0 //!ASSERT_DISABLED
         void* currentPosition = &currentPosition;
         ASSERT(m_origin != m_bound);
         ASSERT(isGrowingDownward()
