Index: Source/cmake/WebKitCompilerFlags.cmake
===================================================================
--- Source/cmake/WebKitCompilerFlags.cmake	(revision 239662)
+++ Source/cmake/WebKitCompilerFlags.cmake	(working copy)
@@ -102,7 +102,7 @@
                                              -Wno-unknown-argument)
     else ()
         WEBKIT_APPEND_GLOBAL_COMPILER_FLAGS(-fno-exceptions)
-        WEBKIT_APPEND_GLOBAL_CXX_FLAGS(-fno-rtti)
+        # WEBKIT_APPEND_GLOBAL_CXX_FLAGS(-fno-rtti)
 
         check_cxx_compiler_flag("-std=c++14" CXX_COMPILER_SUPPORTS_CXX14)
         if (CXX_COMPILER_SUPPORTS_CXX14)
